extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Please place the file under the following directory:
xdm/components/mixins/segment-definition